src/lib/BundleResolver/BundleNameResolver.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\CompatibilityLayer\BundleResolver;
+
+use Ibexa\Bundle\CompatibilityLayer\IbexaCompatibilityLayerBundle;
+use Ibexa\CompatibilityLayer\FullyQualifiedNameResolverInterface;
+
+final class BundleNameResolver
+{
+    /** @var array[] */
```

Comment:
It's better if we use this:
```suggestion
    /** @var array<string, string> */
```

So static analysis does not complain about concatenation for that (line 36):
```php
return $newBundleName . 'Bundle';
```